tkg/clusterclient/clusterclient.go

```diff
@@ -1412,24 +1430,38 @@ func (c *client) kubectlApplyFile(url string) error {
 	if c.currentContext != "" {
 		args = append(args, "--context", c.currentContext)
 	}
-
-	args = append(args, "-f", url)
+	if o.recursive {
+		args = append(args, "--recursive")
+	}
+	if o.namespace != "" {
```

Comment:
Why need to specify the namespace?

Reply:
The TKR manifests don't actually include the namespace but they need to be installed to tkg-system.